services/frontend-service/pkg/cmd/server.go

```diff
@@ -474,3 +488,22 @@ func (p *GrpcProxy) CreateEnvironment(
 	in *api.CreateEnvironmentRequest) (*emptypb.Empty, error) {
 	return p.EnvironmentClient.CreateEnvironment(ctx, in)
 }
+
+func (p *GrpcProxy) StreamStatus(in *api.StreamStatusRequest, stream api.RolloutService_StreamStatusServer) error {
+	if p.RolloutServiceClient == nil {
+		return status.Error(codes.Unimplemented, "rollout status not implemented")
+	}
+	if resp, err := p.RolloutServiceClient.StreamStatus(stream.Context(), in); err != nil {
+		return err
+	} else {
+		for {
+			if item, err := resp.Recv(); err != nil {
```

Comment:
should we add explicit tracing spans here, or is that too much?

Reply:
The underlying instrumentation already does this as for the overview service.